src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const table = document.querySelector('table');
+  const tbody = table.querySelector('tbody');
+  const headers = Array.from(table.querySelectorAll('th'));
+  const rows = Array.from(tbody.querySelectorAll('tr'));
+
+  headers.forEach((header, columnIndex) => {
```

Comment:
use [event delegation ](https://www.freecodecamp.org/news/event-delegation-javascript/) instead of adding a separate event listener to each header